app/src/main/java/org/mozilla/fenix/trackingprotection/TrackingProtectionBlockingFragment.kt

```diff
@@ -19,6 +19,7 @@ class TrackingProtectionBlockingFragment : Fragment(R.layout.fragment_tracking_p
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        category_fingerprinters.isVisible = args.strictMode
         category_tracking_content.isVisible = args.strictMode
```

Comment:
I was thinking that it could be possible for this tracking UI to be updated automatically for stuff like this if the views were programmatically added to the layout.

Could add some sort of `enabledInStrictMode` flag to the tracking protection types and use that to filter based on whether strict mode is enabled.